Client/Ecosia/UI/Impact/Views/MyImpactCell+ViewFactories.swift

```diff
@@ -0,0 +1,187 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+extension MyImpactCell {
+    
+    func makeOutline() -> UIView {
+        let outline = UIView()
+        outline.layer.cornerRadius = UX.Outline.cornerRadius
+        outline.translatesAutoresizingMaskIntoConstraints = false
+        return outline
+    }
+    
+    func makeHowItWorksButton() -> UIControl {
+        let howItWorksButton = UIControl()
+        howItWorksButton.translatesAutoresizingMaskIntoConstraints = false
+        return howItWorksButton
+    }
+    
+    func makeProgressView() -> Progress {
+        Progress(size: UX.ProgressView.progressSize, lineWidth: UX.ProgressView.lineWidth)
+    }
+    
+    func makeIndicatorView() -> Indicator {
+        Indicator(size: UX.ProgressView.progressSize)
+    }
+    
+    func makeTreesIcon() -> UIImageView {
+        let treesIcon = UIImageView()
+        treesIcon.translatesAutoresizingMaskIntoConstraints = false
+        treesIcon.contentMode = .center
+        treesIcon.clipsToBounds = true
+        return treesIcon
+    }
+    
+    func makeTreesCountLabel() -> UILabel {
+        let treesCount = UILabel()
+        treesCount.translatesAutoresizingMaskIntoConstraints = false
+        treesCount.font = .preferredFont(forTextStyle: .title1).bold()
+        treesCount.adjustsFontForContentSizeCategory = true
+        return treesCount
+    }
+    
+    func makeTreesPlantedLabel() -> UILabel {
+        let treesPlanted = UILabel()
+        treesPlanted.translatesAutoresizingMaskIntoConstraints = false
+        treesPlanted.font = .preferredFont(forTextStyle: .body)
+        treesPlanted.adjustsFontForContentSizeCategory = true
+        return treesPlanted
+    }
+    
+    func makeHowItWorksLabel() -> UILabel {
+        let howItWorks = UILabel()
+        howItWorks.translatesAutoresizingMaskIntoConstraints = false
+        howItWorks.font = .preferredFont(forTextStyle: .callout)
+        howItWorks.adjustsFontForContentSizeCategory = true
+        howItWorks.text = .localized(.howItWorks)
+        return howItWorks
+    }
+    
+    func makeHowItWorksIcon() -> UIImageView {
+        let howItWorksIcon = UIImageView()
+        howItWorksIcon.translatesAutoresizingMaskIntoConstraints = false
+        howItWorksIcon.contentMode = .center
+        howItWorksIcon.clipsToBounds = true
+        return howItWorksIcon
+    }    
+}
+
+extension MyImpactCell {
```

Comment:
I am wondering is there any particular reason to have multiple extension sections instead of just one? I assume it's structured by purpose to some extend? If so, would you mind adding // MARK annotations to surface the structure?

Reply:
Sure thing @ecotopian . I was actually meant to add them but then I didn't for some reason 😓 
Will do it now 😉